src/protocols/secure_channel/tests/TestStatusReport.cpp

```diff
@@ -104,6 +105,33 @@ void TestBadStatusReport(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, err != CHIP_NO_ERROR);
 }
 
+void TestMakeBusyStatusReport(nlTestSuite * inSuite, void * inContext)
+{
+    GeneralStatusCode generalCode                 = GeneralStatusCode::kBusy;
+    auto protocolId                               = SecureChannel::Id;
+    uint16_t protocolCode                         = kProtocolCodeBusy;
+    System::Clock::Milliseconds16 minimumWaitTime = System::Clock::Milliseconds16(5000);
+
+    System::PacketBufferHandle handle = StatusReport::MakeBusyStatusReportMessage(minimumWaitTime);
+    NL_TEST_ASSERT(inSuite, !handle.IsNull());
+
+    StatusReport reportToParse;
+    CHIP_ERROR err = reportToParse.Parse(std::move(handle));
+    NL_TEST_ASSERT(inSuite, CHIP_NO_ERROR == err);
+    NL_TEST_ASSERT(inSuite, reportToParse.GetGeneralCode() == generalCode);
+    NL_TEST_ASSERT(inSuite, reportToParse.GetProtocolId() == protocolId);
+    NL_TEST_ASSERT(inSuite, reportToParse.GetProtocolCode() == protocolCode);
+
+    const System::PacketBufferHandle & rcvData = reportToParse.GetProtocolData();
+    NL_TEST_ASSERT(inSuite, !rcvData.IsNull());
+    NL_TEST_ASSERT(inSuite, rcvData->DataLength() == sizeof(minimumWaitTime));
+
+    uint16_t readMinimumWaitTime = 0;
+    Encoding::LittleEndian::Reader reader(rcvData->Start(), rcvData->DataLength());
+    NL_TEST_ASSERT(inSuite, CHIP_NO_ERROR == reader.Read16(&readMinimumWaitTime).StatusCode());
```

Comment:
Based on prior comment, for follow-up:
- Being able to parse the minimum wait time would benefit to be done in SatusReport module.